helios-services/src/main/java/com/spotify/helios/servicescommon/statistics/NoopZooKeeperMetrics.java

```diff
@@ -17,7 +17,14 @@
 
 package com.spotify.helios.servicescommon.statistics;
 
+import com.codahale.metrics.Timer;
+
 public class NoopZooKeeperMetrics implements ZooKeeperMetrics {
   @Override
   public void zookeeperTransientError() {}
+
+  @Override
+  public Timer timer(String name) {
+    return new Timer();
```

Comment:
It'd be nice to actually have a no op, but the nice thing about the current interface is that a Timer takes care of timing the duration. It's a bit more unwieldy if the caller has to handle timing a Callable when the Timer class already does this.